lib/movie_details.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'styles.dart';
+
+import 'movie.dart';
+
+class MovieDetails extends StatelessWidget {
+  MovieDetails({
+    super.key,
+    required this.movie,
+  });
+
+  final Movie movie;
+
+  TextStyle descriptionText = const TextStyle(
+    fontSize: fontText,
+    color: Colors.white54,
+  );
+
+  static const int maxLinesOverviewText = 10;
+
+  static const double paddingTopBetweenDescriptionOverview = 12;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: double.infinity,
+      child: ListView(
+        shrinkWrap: true,
+        physics: const NeverScrollableScrollPhysics(),
+        children: [
+          const Center(
+            child: Text("Description:", style: titles),
```

Comment:
you can move the hardcoded text into his own constant and missing a trailing comma

Reply:
Done